src/processSymbolLayer.ts

```diff
@@ -139,6 +139,23 @@ const formatLineSymbolizer = (
   if (layer.markerPlacement.type === "CIMMarkerPlacementAlongLineSameSize") {
     const size = ptToPxProp(layer, "size", 10);
     const template = processMarkerPlacementAlongLine(markerPlacement, size);
+    if (symbolizer.kind === "Mark") {
+      const marker = symbolizer as MarkSymbolizer;
+      if (!marker?.wellKnownName?.startsWith("wkt://") && marker.strokeWidth === 0) {
+        // If the marker has a known shape with strokeWidth of 0, we don't want to render it because Geoserver still draws a line.
+        symbolizer.strokeWidth = undefined;
+        symbolizer.strokeColor = undefined;
+        symbolizer.strokeOpacity = undefined;
```

Comment:
Can you try with:

```suggestion
        delete symbolizer.strokeWidth
        delete symbolizer.strokeColor
        delete symbolizer.strokeOpacity
```
That would be cleaner.

Reply:
I've done a PR to correct that here : https://github.com/geostyler/geostyler-lyrx-parser/pull/126 (that target your branch)